### PR TITLE
Added docs for Jenkin pipeline

### PR DESCRIPTION
**Pull request by shhegd-msft**

### CODE REVIEW

File: 03-Security-In-CICD/Readme.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
-# Security Verification Tests (SVTs) in CICD Pipeline
+
+# Security Verification Tests (SVTs)
 
 ![Security_In_CICD](../Images/Security_In_CICD.jpg)
 
 > **Notes** 
 > - We also plan to support miniaturized ARM templates which can be combined with existing templates and, subsequently, to analyze 'ARM template equivalence' from a security standpoint.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: Not sure if we really want to claim on mini arm templates

File: 03-Security-In-CICD/Readme.md [hunk 2]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
 ![03_AzSDK_Logs](../Images/03_AzSDK_Logs.png)
 
 [Back to top...](Readme.md#contents)
+
+# Security Verification Tests (SVTs) in Jenkins pipeline
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: should we call it as preview ?

File: 03-Security-In-CICD/Readme.md [hunk 3]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
...
+	
+> Note : AzSDK Plugin requires PowerShell to be present on Jenkins Server. Therefore, the Plugin is currently supported for Windows machines only.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: the Plugin  => the plugin

Comment (sbyna-msft) [reply 03-Security-In-CICD/Readme.md]: Many places Plugin is with caps P, i think it should be small

File: 03-Security-In-CICD/Readme.md [hunk 4]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
...
+
+### Enable AzSDK extension for your Jenkins
+	
+Currently AzSDK CICD extension/Plugin has not been published on Jenkins repository, However you can use Jenkins Web UI to upload this Plugin([AzSDK_CICD_Jenkins_Plugin.hpi](Assets/AzSDK_CICD_Jenkins_Plugin.hpi) file) to Jenkins or place it in '$JENKINS_HOME/Plugins' location.
+	
+**Step to upload Plugin using Jenkins Web UI**
+	
+ Go to Home Page --> Manage Jenkins --> Manage Plugins -->  Select Advanced --> Upload Plugin file "[AzSDK_CICD_Jenkins_Plugin.hpi](Assets/AzSDK_CICD_Jenkins_Plugin.hpi)"
+
+![03_Upload_Plugin](../Images/03_Upload_Plugin.png)
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: these images are not coming

File: 03-Security-In-CICD/Readme.md [hunk 5]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
...
+
+![03_Upload_Plugin](../Images/03_Upload_Plugin.png)  
+ 
+![03_Install_Plugin](../Images/03_Install_Plugin.png)
+ <br/>Plugin is successfully imported. Now let's use Plugin to scan Azure Resources.  
+
+[Back to top...](Readme.md#contents)
+
+### Walkthrough
+This part assumes that you are familiar with Jenkins pipeline at a basic level. To get the basics and setup Jenkins you can refer [article](https://jenkins.io/doc/).
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: To get the basics and setup Jenkins you can refer [article](https://jenkins.io/doc/).
  
May be "To explore more on Jenkins, refer: ----------------"

File: 03-Security-In-CICD/Readme.md [hunk 6]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
...
+
+### Adding SVTs in the Jenkins pipeline
+	
+- #### Step-1: Configure Service Principal (SPN) credentials
+	To run the SVT, AzSDK need SPN/application which has reader on resource group.
+	To create SPN and assign role, refer [link](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal).
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: To create SPN and assign role, refer

the above statement requires re-work

File: 03-Security-In-CICD/Readme.md [hunk 7]
```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
...
+	![03_Save_Job](../Images/03_Save_Job.png)
+[Back to top...](Readme.md#contents)
+### Verifying that the SVTs have been added and configured correctly
+**Step-1:** Trigger the build.
+	![03_Trigger_Build_1](../Images/03_Trigger_Build_1.png)  
+**Step-2:** Verify that the build has started.
+	![03_Trigger_Build_2](../Images/03_Trigger_Build_2.png)	  
+**Step-3:** View the 'Console Output'.
+	![03_Trigger_Build_3](../Images/03_Trigger_Build_3.png)	  
+**Step-4:** See the summary "CSV" and detailed "LOG" output files for the AzSDK SVTs.
```
Comment (sbyna-msft) [inline 03-Security-In-CICD/Readme.md]: steps are not uniform with others

Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: Resolved.

File: 03-Security-In-CICD/Readme.md (hunk 5 above)
Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: Resolved.

File: 03-Security-In-CICD/Readme.md (hunk 4 above)
Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: Image is coming for me. 
All images are require border formatting.

File: 03-Security-In-CICD/Readme.md (hunk 3 above)
Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: Resolved

File: 03-Security-In-CICD/Readme.md (hunk 2 above)
Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: Resolved. Added as Preview

File: 03-Security-In-CICD/Readme.md (hunk 1 above)
Comment (azsdksupext-vh-msft) [reply 03-Security-In-CICD/Readme.md]: For now I have removed this text.

**Pull request merged**